addon/utils/cookies.js

```diff
@@ -0,0 +1,31 @@
+let Cookies = {
+  create(name, value, days) {
+    var expires;
+
+    if (days) {
+      var date = new Date();
+      date.setTime(date.getTime() + (days * 24 * 60 * 60 * 1000));
+      expires = "; expires=" + date.toGMTString();
+    } else {
+      expires = "";
+    }
+    document.cookie = encodeURIComponent(name) + "=" + encodeURIComponent(value) + expires + "; path=/";
```

Comment:
Yeah, I think so — if a domain is not specified it will default to the document.location.host, which will be overly specific (i.e. "dashboard.aptible.com" when we want the cookie to be for "aptible.com"). According to https://developer.mozilla.org/en-US/docs/Web/API/Document/cookie